frontend/src/pages/premium/waitlist.test.tsx

```diff
@@ -1,16 +1,44 @@
 import React from "react";
-import { act, render } from "@testing-library/react";
+import { act, render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
 import { axe } from "jest-axe";
 import { mockFluentReact } from "../../../__mocks__/modules/fluent__react";
 import { mockNextRouter } from "../../../__mocks__/modules/next__router";
 import { mockConfigModule } from "../../../__mocks__/configMock";
+import { setMockRuntimeData } from "../../../__mocks__/hooks/api/runtimeData";
+import { setMockUserData } from "../../../__mocks__/hooks/api/user";
+import { setMockProfileData } from "../../../__mocks__/hooks/api/profile";
 
 import PremiumWaitlist from "./waitlist.page";
 
 jest.mock("@fluent/react", () => mockFluentReact);
 jest.mock("next/router", () => mockNextRouter);
 jest.mock("../../config.ts", () => mockConfigModule);
 jest.mock("../../hooks/gaViewPing.ts");
+jest.mock("../../components/waitlist/countryPicker.tsx", () => ({
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  CountryPicker: (props: any) => <select {...props} />,
```

Comment:
nit/question: Why the use of `any` here?

Reply:
I love that you're asking that question! And in fact, when setting up the mocks I figured I'd be ignoring most of the usually-required props, but it's actually fairly easy to use the right props here, so I did: 0ae82355